### ISSUE TITLE
Game freezes instead of crash, no DMP or log available

### ISSUE THREAD

**Issue by TheOneWhoseUsernameWasTaken** [Warpten/hzdnptr#2]
First, thanks for this project. It's the only one that actually seems to do anything for "Horizon Zero Stability" on my system.

Unfortunately, there are some instances where the game previously crashed and it now only freezes. I then have to kill it via task manager. Since it didn't really crash, there's no corresponding DMP file available, and the last entry in the patchdll.log is from 5-10 minutes before the freeze. The Windows event log records an error bucket type 0 event with the name "D3DDRED2" and the following problem signature:
```
P1: VEN_10DE
P2: 27.21.14.5167
P3: 0x887A0006
P4: 3
P5: HorizonZeroDawn.exe
P6: Unknown
P7: 
P8: 
P9: 
P10: 
```
There is no further information (the event log mentions that files have been saved to certain locations, but those are actually empty).

It seems to point to a problem with DirectX, but I have already made a clean reinstall of the most recent drivers for my GPU (GTX 1060).

Is there another way I can still provide some useful debugging information?

**Comment by Warpten** [comment]
Yeah I've run into these today (two occurrences). This is going to be a hard one to crack...

**Comment by Warpten** [comment]
Please try with e6fba1b6bcfe07. It won't improve things, but may make the game crash again instead of locking up.

**Comment by TheOneWhoseUsernameWasTaken** [comment]
I tried it. At least I think I did. The patch.dll had an SHA-1 of 058e302e8994c8c9a59975da90861c65b4bf335d.

The game still freezes instead of crashing. The only noticeable difference is that now, there's also no entry in the Windows event log that corresponds to the freeze.

**Comment by Warpten** [comment]
Yep, that SHA-1 looks correct.

I don't know what to tell you. I'm running it on my system. I got a crash, but no freeze yet.

**Comment by TheOneWhoseUsernameWasTaken** [comment]
No worries. I mean, sure, I'd like to play the game for longer than an hour (tops) without having to repeat everything since the last savepoint (who needs free saving, eh?). But it's not like it's your fault. You're just trying to clean up somebody else's mess as best you can.

**Comment by TheOneWhoseUsernameWasTaken** [comment]
It would seem so. I had already finished the game, so the urgency is gone. But now I had some spare time, started a new game with 1.0.2 and played for a couple of hours. No crash so far. Whether it's the patch, or just luck, is anyone's guess...
